spec/spec_helper.rb

```diff
@@ -1,15 +1,19 @@
 begin
   require 'simplecov'
-  require 'codeclimate-test-reporter'
+  require 'codacy-coverage'
+
+  SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
+    SimpleCov::Formatter::HTMLFormatter,
+    Codacy::Formatter
+  ]
 
   SimpleCov.start do
     add_filter '/spec/'
-
-    formatter SimpleCov::Formatter::MultiFormatter.new([
-      SimpleCov::Formatter::HTMLFormatter,
-      CodeClimate::TestReporter::Formatter
-    ])
   end
+
+  # Skip Codacy when running locally, to display the Simplecov summary in the console
+  # and write an updated coverage/index.html
+  Codacy::Reporter.start if Codacy::Formatter.new.send :should_run?
```

Comment:
Codacy's `should_run?` logic detects whether it is in a local or CI environment, and Codacy does not send data when local. Unfortunately, it does still block Simplecov from writing its local output. So here we ensure Codacy doesn't interfere with Simplecov when running local.